src/main/java/org/frc5687/deepspace/robot/utils/Gamepad.java

```diff
@@ -0,0 +1,90 @@
+package org.frc5687.switchbot.robot.utils;
+
```

Comment:
When you copy a file betwen projects, you have to correct the package name.

Reply:
(replace switchbot with deepspace)

Reply:
Oh, I am a fool. 
My bad.